### DISCUSSION TITLE
Xamarin Native

### DISCUSSION THREAD

**Discussion by MikhailFrenkel** [dotnet/maui#306] [Q&A] [answered]
Hi everyone,
Will Xamarin.Native be supported after release .NET MAUI?

**Comment by sacdevops** [comment, marked as answer]
You might not like the answer here, which is why I try to explain it to you slowly and emotionally so that nobody is hurt. To be honest, _yes_... 😂  It will be Called .NET for iOS and .NET for Android.

**Reply by WashingtonARamos** [reply]
.NET for Android and .NET for iOS sounds much better than Xamarin.Android and Xamarin.iOS or Xamarin.Native.
